release-process-documentation/release-team-guides/ci-signal-playbook.md

```diff
@@ -74,27 +174,26 @@ Examples:
 - [[e2e failure] [sig-network] Networking Granular Checks: Services [Slow] should function for client IP based session affinity: udp](https://github.com/kubernetes/kubernetes/issues/54524)
 - [[e2e failure] [sig-network] Networking Granular Checks: Services [Slow] should function for client IP based session affinity: http](https://github.com/kubernetes/kubernetes/issues/54571)
 
-
-### Reporting Status
-
-I'm not sure I want to act as a human dashboard generator on an ongoing basis, so I'd like to find a better way to do this.
-
-I have been updating the sig-release issue for the latest release each time I scrub through dashboards, eg: [1.9.0-alpha.2 issue](https://github.com/kubernetes/sig-release/issues/22#issuecomment-340970138)
-
-## Code Slush
-
-I check [sig-release-master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking) and [sig-release-master-upgrade](https://k8s-testgrid.appspot.com/sig-release-master-upgrade) daily.
-
-I will ocasionally check [sig-release-1.y-blocking](https://k8s-testgrid.appspot.com/sig-release-1.y-blocking) and work with the Test Infra role to make sure the list of jobs looks correct
-
-## Code Freeze
-
-I check [sig-release-master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking) and [sig-release-master-upgrade](https://k8s-testgrid.appspot.com/sig-release-master-upgrade) daily.
-
-I expect more responsiveness from SIG's at this point. In addition to filing github issues, I start pinging issues for updates, and pinging sig slack channels.
-
-## Exit Code Freeze
-
-I stop looking at release-master-blocking
-
-I look solely at release-1.y-blocking and release-master-upgrade
+### Monitoring Commits for test failure triangulation
```

Comment:
This section too could be duplicated into the release lead and bug triage docs.

Reply:
I'm a fan of minimizing the duplication, if possible. If this is valid for RT Lead & Bug Triage, let's have those playbooks link back to this.